experimental/dds/sequence-deprecated/package.json

```diff
@@ -92,9 +132,20 @@
 		"moment": "^2.21.0",
 		"prettier": "~3.0.3",
 		"rimraf": "^4.4.0",
-		"tsc-multi": "^1.1.0",
 		"typescript": "~5.1.6"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"build:docs": {
+				"dependsOn": [
+					"...",
+					"api-extractor:commonjs",
+					"api-extractor:esnext"
+				],
+				"script": false
+			}
+		}
+	},
```

Comment:
It is.  The global config is set up for 'build:docs' to invoke api-extractor directly (i.e., only depends on 'tsc').

If I remove it:
ERROR: @fluid-experimental/sequence-deprecated: Unable to find dependent 'build:docs'

I debugged this earlier this week.  IIRC, it translates to "somehow the task graph you built doesn't include the 'build:docs' task prequestites.'

That's when I noticed everyone else using '"build:docs": "fluid-build . --task api"' has this local config, which I believe is temporary until we have everyone on the "api" plan.

Reply:
That end result doesn't make sense to me.
What this is saying is that before `build:docs` can "run" (doesn't run because there is no script according to the `"script": false` config even though there is a script entry), those two api-extractor tasks need to run.
If the script were run, it would execute those two api-extractor tasks because they are steps in the `api` task.
If there were going to be a dependency, I don't know why it wouldn't be exactly `api`.
But this does appear to be what Tyler was doing as he converted. So okay.
Maybe the end is that `build:docs` entries just completely go away.